Gems/Atom/RPI/Code/Source/RPI.Private/RPISystemComponent.h

```diff
@@ -49,6 +51,12 @@ namespace AZ
             void Activate() override;
             void Deactivate() override;
 
+            ///////////////////////////////////////////////////////////////////
+            //! IXRRegisterInterface
+            void RegisterXRInterface(XRRenderingInterface* xrSystemInterface) override;
+            void UnRegisterXRInterface() override;
+            ///////////////////////////////////////////////////////////////////
```

Comment:
nit: I think the common way to indicate overrides is

````
// IXRRegisterInterface overrides...
````

Up to you if you want to do this change. I find the full ////// lines a bit too much. Also is shouldn't be a doxygen comment `//!`.